docs/UserGuide.md

```diff
@@ -265,17 +328,17 @@ Format: `i-restock`
 
 Adds an employee to the contact list.
 
-Format: `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT a/ADDRESS [t/TAG] ...`
+Format: `c-add n/NAME p/PHONE_NUMBER e/EMERGENCY_CONTACT [t/TAG]…​`
 
-<div markdown="span" class="alert alert-primary">:bulb: 
+<div markdown="span" class="alert alert-primary">:bulb:
 
 **Tip:**
 An employee can have any number of tags (including 0)
 </div>
 
 Examples:
-* `c-add n/John Doe p/98765432 e/81234567 a/Blk 123 ABC Road`.
-* `c-add n/Betsy Crowe e/81234567 p/89007413 a/Blk 120 ABC Road t/Friday t/PartTime`.
+* `c-add n/John Doe p/98765432 e/81234567`.
+* `c-add n/Betsy Crowe e/81234567 p/1234567 t/morning shift t/part-time`.
```

Comment:
for line 340 ~ 341, A similar problem here, you may want to change back to what it was. Thanks